tests/unit/worker/test_submission_worker.py

```diff
@@ -45,6 +123,18 @@ def test_return_file_url_per_environment(self):
         returned_url = return_file_url_per_environment(self.url)
         self.assertEqual(returned_url, "http://testserver/test/url")
 
+    @mock.patch("scripts.workers.submission_worker.CHALLENGE_DATA_DIR", new="/tmp/test-dir/compute/challenge_data/challenge_{challenge_id}")
+    @mock.patch("scripts.workers.submission_worker.importlib.import_module", return_value="challenge_module")
+    @mock.patch("scripts.workers.submission_worker.download_and_extract_zip_file")
+    def test_extract_submission_data(self, mocked_download_and_extract_zip_file, mocked_import_module):
+        phases = self.challenge.challengephase_set.all()
+        extract_challenge_data(self.challenge, phases)
+        evaluation_script_url = "http://testserver{}".format(self.challenge.evaluation_script.url)
```

Comment:
can we create a common variable for `http://testserver`, hardcoding it at multiple places in a file is a bad practice.

Reply:
Okay right. Made changes, please see new commit.

Reply:
@nikochiko can you please fix the reviews in your previous PR? we need to merge it before getting to this PR as it has redundant code

Reply:
That pull is almost done as well. Made some changes here to update it with base changes in previous PR like changing `unittest.TestCase` with `rest_framework.APITestCase`.